tests/genomics_tests/test_genomic_pipeline.py

```diff
@@ -3253,7 +3253,7 @@ def test_cvl_w2_manifest_ingestion(self):
 
         members = self.member_dao.get_all()
         for m in members:
-            self.member_dao.update_member_state(m, GenomicWorkflowState.W1)
+            self.member_dao.update_member_workflow_state(m, GenomicWorkflowState.W1)
```

Comment:
It doesn't need to be in your PR, but we should clean all those old 2020 CVL tests out.

Reply:
Yea was thinking about adding a PR to remove all the previous CVL workflows since so much has changed and would probably help alleviate any confusion for the future. RIP previous CVL wfs